plaidml/BUILD

```diff
@@ -176,10 +176,24 @@ string_flag(
 
 string_flag(
     name = "target",
-    values = ["llvm_cpu", "foo"],
+    values = ["llvm_cpu", "intel_gen"],
```

Comment:
Perhaps we should leave this as being an open-set. In fact this will be a requirement soon.

Reply:
Done